mule-extensions-api/src/main/java/org/mule/runtime/extension/api/ExtensionConstants.java

```diff
@@ -152,6 +157,8 @@
   public static final String SOURCE_TRANSACTIONAL_ACTION_PARAMETER_DESCRIPTION =
       "The type of beginning action that sources can take regarding transactions.";
 
+  public static final String SOURCE_TRANSACTIN_TYPE_PARAMETER_DESCRIPTION = "The type of transaction to use";
```

Comment:
TRANSACTN -> TRANSACTION. Define alongside the name parameter. Why is this specific to Source? The other one isnt' and this seems to apply to any component capable of starting a tx in the future